docs/4. Product Features/06.  HSM Templates.md

```
@@ -33,142 +33,108 @@ Session Window - A session window refers to the 24-hour period following a user'
-<img width="213" alt="image" src="https://github.com/glific/docs/assets/122982753/5038931a-5b99-4642-addb-3fb9c452c4fb"/>
-<br />
+
 2. Click on `+ CREATE` button from the top right corner of Templates page.
 
-<img width="537" height="312" alt="image" src="https://github.com/user-attachments/assets/05d5ef7d-41c4-4bbf-8368-1e425d0880ff" />
+<img width="" height="833" alt="Screenshot 2026-07-24 at 12 29 30 PM" src="https://github.com/user-attachments/assets/6e28bacd-e794-4cfb-9216-4f283f68ac01" />
 
-<br />
-<br />
 
 3. `Add A New HSM Template` page opens up.
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)